app/javascript/actionsPanel/actionsPanel.js

```diff
@@ -345,18 +345,16 @@ export function addAdjustTagListeners() {
     },
   );
 
-  const adjustTagSubmitBtn = document.getElementById('tag-adjust-submit');
-  if (adjustTagSubmitBtn) {
-    adjustTagSubmitBtn.addEventListener('click', (e) => {
+  const form = document.getElementById('tag-adjust-submit')?.form;
+  if (form) {
+    form.addEventListener('submit', (e) => {
```

Comment:
The reason the HTML validation wasn't working was because the original code was preventing the default action on the submit button which mean the form submit event never fired which triggers the HTML validation.